### ISSUE TITLE
nn.R

### ISSUE THREAD

**Issue by amidisturbing** [amidisturbing/ml2-breast-cancer#3]
ConfusionMatrix looking like this:
         0   1
  0 286 170
  1   0   0

but as far as I can see there were classes 0 being predicted
<img width="255" alt="Bildschirmfoto 2021-02-27 um 16 56 28" src="https://user-images.githubusercontent.com/28503640/109392579-2b283580-791d-11eb-866b-2db0c8dfe752.png">

**Comment by s81320** [comment]
Model predicts dominant class for all data points. That tells us the model is not performing well, but it's not really false or a strange error: At least it is predicting the dominant class :-)
We should check the activation (ReLu by default? other options: softmax) and loss functions (any defaults?).
